src/tram/views.py

```diff
@@ -162,7 +176,10 @@ def upload(request):
     else:
         return HttpResponseBadRequest("Unsupported file type")
 
-    return HttpResponse("File saved for processing", status=200)
+    if dpj:
+        headers.update({'job-id': dpj.pk, 'doc-id': dpj.document.pk})
+
+    return HttpResponse("File saved for processing", headers=headers, status=200)
```

Comment:
Rather than returning these values in non-standard headers, please return them in the response body as a JSON object. (I.e. change HttpResponse to JsonResponse, add fields for the "File saved for processing" message, doc ID, and job ID.)